src/nvme/types.h

```diff
@@ -2785,7 +2787,9 @@ struct nvme_telemetry_log {
 	__le16	dalb1;
 	__le16	dalb2;
 	__le16	dalb3;
-	__u8	rsvd14[368];
+	__u8    rsvd14[2];
+	__le32  dalb4;
+	__u8    rsvd20[362];
 	__u8	ctrlavail;
```

Comment:
All good, but where's the plugin?
Shouldn't we modify the commit message to refer to the changes we actually did?

Reply:
The plugin part is in the nvme-cli PR (https://github.com/linux-nvme/nvme-cli/pull/1223):

https://github.com/linux-nvme/nvme-cli/pull/1223/commits/d361c3781778e658fb1f67109847ebf53111ecd9

Reply:
Ah got it, I changed the commit message title. Indeed a bit confusing.